accounts/ethstore/src/accounts_dir/disk.rs

```diff
@@ -141,28 +141,32 @@ impl RootDiskDirectory {
 impl<T> DiskDirectory<T> where T: KeyFileManager {
 	/// Create new disk directory instance
 	pub fn new<P>(path: P, key_manager: T) -> Self where P: AsRef<Path> {
-		DiskDirectory {
+		Self {
 			path: path.as_ref().to_path_buf(),
-			key_manager: key_manager,
+			key_manager,
 		}
 	}
 
 	fn files(&self) -> Result<Vec<PathBuf>, Error>  {
 		Ok(fs::read_dir(&self.path)?
-			.flat_map(Result::ok)
-			.filter(|entry| {
-				let metadata = entry.metadata().ok();
-				let file_name = entry.file_name();
-				let name = file_name.to_string_lossy();
-				// filter directories
-				metadata.map_or(false, |m| !m.is_dir()) &&
-					// hidden files
-					!name.starts_with(".") &&
-					// other ignored files
-					!IGNORED_FILES.contains(&&*name)
+			.filter_map(|entry| {
+				if let Ok(entry) = entry {
+					let metadata = entry.metadata().ok();
+					let file_name = entry.file_name();
+					let name = file_name.to_string_lossy();
+					// filter directories
+					if metadata.map_or(false, |m| !m.is_dir()) &&
+						// hidden files
+						!name.starts_with('.') &&
+						// other ignored files
+						!IGNORED_FILES.contains(&&*name) {
```

Comment:
it would be more readable `{` on the next line, the indentation confuses me a bit

Reply:
done